python/axipcie/_AxiGpuAsyncCore.py

```diff
@@ -151,53 +151,72 @@ def __init__(self,
             pollInterval = 1,
         ))
 
+        self.add(pr.RemoteVariable(
+            name         = 'DynamicRouteMasks',
+            offset       = 0x02C,
+            bitSize      = 8,
+            disp         = '{}',
+            mode         = 'RW',
+            pollInterval = 1,
```

Comment:
@mmishra9 Remove `pollInterval = 1,` because this is a RW variables

Reply:
Done